### PR TITLE
Add privacy and terms of service links.

### PR DESCRIPTION
**Pull request by andresusanopinto**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -60,6 +60,11 @@
   <!-- Placeholder for dynamically generated test suites. -->
   <div id="content"></div>
 
+  <footer>
+    <span> <a href="//www.google.com/accounts/TOS">Terms</a></span>
```
Comment (KaptenJansson) [inline src/index.html]: Nit Remove space.

**Comment by KaptenJansson** [comment]
LGTM with nit.

File: src/index.html (hunk 1 above)
Comment (andresusanopinto) [inline src/index.html]: Done.

**Comment by andresusanopinto** [comment]
Can you merge and push?

**Comment by KaptenJansson** [comment]
This is now pushed and served.

**Pull request merged**